eng/DownloadSharedSource.ps1

```diff
@@ -1,11 +1,17 @@
-$downloadPath = Resolve-Path (Join-Path $PSScriptRoot '..' 'src' 'assets' 'Azure.Core.Shared')
-$files = "ClientDiagnostics.cs", "ArrayBufferWriter.cs", "DiagnosticScope.cs"
-$baseUrl = "https://raw.githubusercontent.com/Azure/azure-sdk-for-net/master/sdk/core/Azure.Core/src/Shared/"
-
-foreach ($file in $files)
+function Invoke-DownloadSharedSource
```

Comment:
It'ss impossible to invoke this script independently now.

Reply:
Didn't know that was a requirement. Will fix.